code/gui/griddisplay.cpp

```diff
@@ -107,5 +140,30 @@ void GridDisplay::drawGrid() {
 
 void GridDisplay::hideGrid() {
     //TODO
-    //gridDisplayed = false;
+    //hide or disable qgraphicsview
+    //gridDisplayed = true;
+}
+
+void GridDisplay::selectRobotPosition(QString weightSelected) {
+    if (weightSelected == "Start Configuration") {
+        startPosSelected = true;
+        endPosSelected = false;
+    } else if (weightSelected == "End Configuration") {
```

Comment:
I'm not a huge fan of using strings to represent state; it'd be better to define an `enum` instead and set the text (if the text is being used somewhere) based on the enum